components/OrchestratorList/index.tsx

```diff
@@ -158,6 +168,7 @@ const OrchestratorList = ({
 
             rewardCallRatio,
             rewardCut: Number(row.rewardCut) / 1000000,
+            treasuryRewardCut: Number(treasuryRewardCutRate / BigInt(1e18)) / 1e9,
```

Comment:
I wasn't sure what unit this value was returned, but this conversion seems to work. There, however, might be ways to improve this calculation.